pyof/v0x01/foundation/basic_types.py

```diff
@@ -201,6 +201,14 @@ def get_size(self):
         return 6
 
 
+def _int2hex(n):
```

Comment:
This should be inside the method unpack() or a `util.py` file. I will accept the PR but, please open an issue for that.